pyproj/_transformer.pyx

```diff
@@ -30,38 +45,41 @@ cdef class _Transformer:
         if self.projctx is not NULL:
             proj_context_destroy(self.projctx)
 
-    def set_radians_io(self):
-        self.input_radians = proj_angular_input(self.projpj, PJ_FWD)
-        self.output_radians = proj_angular_output(self.projpj, PJ_FWD)
+    def _set_radians_io(self):
+        self.input_radians = {
```

Comment:
those are not public right? (since this in the cython class) 
(just to be sure, as I don't think they should be in their current form)

Reply:
Yep, confirmed this myself :)

Reply:
I think making that private is a good idea. I made the change.